### DISCUSSION TITLE
Add owner feature

### DISCUSSION THREAD

**Discussion by FizzyMUC** [LeoKlaus/Paperparrot#112] [Feature Requests]
Since version 1.14 Paperless offers a feature for ownership of documents, allowing for multiple users. 
The owner cannot be edited in the app. Can this be implemented shortly?
In my instance I'm using storage paths depending on the respective owner. Would be helpful to be able to change/update in the app.

**Comment by LeoKlaus** [comment]
Hi FizzyMUC,

thank you for the suggestion.
I've had this on my list for quite some time now and am working on it already. I'll let you know once I have news!

**Comment by LeoKlaus** [comment]
Oh, forgot to close this. Permissions are supported in Paperparrot now :)
